docusaurus/docs/React/components/message-components/attachment/voice-recording.mdx

```diff
@@ -0,0 +1,235 @@
+import VoiceRecordingPlayer from "../../../assets/voice-recording-player.png";
+import VoiceRecordingPlayerInProgress from "../../../assets/voice-recording-player-in-progress.png";
+import VoiceRecordingPlayerStoppedRepro from "../../../assets/voice-recording-player-stopped-repro.png";
+import VoiceRecordingPlayerFileSizeFallback from "../../../assets/voice-recording-player-file-size-fallback.png";
+import VoiceRecordingPlayerFallbackTitle from "../../../assets/voice-recording-fallback-title.png";
+import VoiceRecordingPlayerMissingWaveformData from "../../../assets/voice-recording-empty-waveform-data.png";
+
+import QuotedVoiceRecording from "../../../assets/voice-recording-quoted.png";
+import QuotedVoiceRecordingFileSizeFallback from "../../../assets/voice-recording-quoted-file-size-fallback.png";
+import QuotedVoiceRecordingFallbackTitle from "../../../assets/voice-recording-quoted-fallback-title.png";
+
+Audio attachments recorded directly from the chat UI are called voice recordings. The SDK provides a default implementation called <GHComponentLink text='VoiceRecording' path='/Attachment/VoiceRecording.tsx'/>. The default component renders or <GHComponentLink text='VoiceRecordingPlayer component' path='/Attachment/VoiceRecording.tsx'/> or <GHComponentLink text='QuotedVoiceRecording' path='/Attachment/VoiceRecording.tsx'/>.
+
+The `VoiceRecordingPlayer` component is displayed in the message attachment list and is used to reproduce the audio.
+
+<ImageShowcase
+    border
+    items={[
+        {
+            image: VoiceRecordingPlayer,
+            caption: <span>VoiceRecordingPlayer component</span>,
+            alt: 'Image of the VoiceRecordingPlayer component',
+        },
+    ]}
+/>
+
+Whereas `QuotedVoiceRecording` is used to display basic information about the voice recording in quoted message reply.
+
+<ImageShowcase
+    border
+    items={[
+        {
+            image: QuotedVoiceRecording,
+            caption: <span>QuotedVoiceRecording component</span>,
+            alt: 'Image of the QuotedVoiceRecording component',
+        },
+    ]}
+/>
+
+## Attachment payload
+
+The response payload for the voice recording attachment comes with the following properties:
+
+<ImageShowcase
+    border
+    items={[
+        {
+            image: VoiceRecordingRequestPayload,
+            caption: <span>Voice recording payload</span>,
+            alt: 'Image of the voice recording payload',
+        },
+    ]}
+/>
+
+These properties serve the following purpose:
+
+| Property          | Description                                                                                                                                                                                                     |
+|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
I think your formatting might be configured improperly - I get a ton of formatting changes when I run it on my machine.

Reply:
fixed